python-avd/pyavd/_eos_cli_config_gen/j2templates/documentation/monitoring.j2

```diff
@@ -62,4 +62,6 @@
 {%     include 'documentation/trackers.j2' %}
 {## Monitor Telemetry Postcard #}
 {%     include 'documentation/monitor-telemetry-postcard-policy.j2' %}
+{## Monitor Server Radius #}
```

Comment:
`if` statement on lines 18-43 needs to be adjusted to match if `monitor_server_radius` is defined. Right now `host2`'s documentation is missing section for `Monitor Server Radius Summary`

Reply:
Fixed. Thanks